test.txt

```diff
@@ -0,0 +1 @@
+Test push commit to the remote in the workflow
```

Comment:
This can be deleted!

Reply:
Interesting! I deleted the file see https://github.com/brown-bnc/xnat-tools/pull/47/commits/9748d65c2dbf6978e1febada490bebe22921545e

So I think I must have something on CI